### DISCUSSION TITLE
Bug with user profile memory line thing.

### DISCUSSION THREAD

**Discussion by HreinnHugur** [valderosch/atom-cloud-storage#20] [Q&A] [answered]
Aftere deleting large files from cloud storage the memory line doesn't show the correct value of memory left in cloud. How to fix ?

**Comment by valderosch** [comment, marked as answer]
Thank you fow giving feedback about this bug.
Now application works in test mode with locked value of user storage size (1GB). We recoment not to upload large files right now.
About your problem, we will fix this in next app update.
